package.json

```diff
@@ -1,24 +1,22 @@
 {
   "name": "@hashicorp/design-system",
-  "private": true,
   "version": "0.0.0",
-  "workspaces": [
-    "./packages/*",
-    "./showcase",
-    "./website"
-  ],
+  "private": true,
+  "scripts": {
+    "release-packages": "pnpm changeset publish",
+    "version-packages": "pnpm changeset version && pnpm install --mode update-lockfile && pnpm -F website generate-changelog-markdown-files"
+  },
   "devDependencies": {
     "@changesets/cli": "^2.27.11",
     "@changesets/get-github-info": "^0.6.0",
+    "@glint/core": "^1.5.1",
     "dotenv": "^16.4.7"
   },
-  "scripts": {
-    "release-packages": "yarn changeset publish",
-    "version-packages": "yarn changeset version && yarn install --mode update-lockfile && yarn workspace website generate-changelog-markdown-files"
-  },
-  "packageManager": "yarn@4.5.3",
-  "resolutions": {
-    "broccoli-asset-rewrite@^2.0.0": "patch:broccoli-asset-rewrite@npm%3A2.0.0#./.yarn/patches/broccoli-asset-rewrite-npm-2.0.0-c4ce42084a.patch",
```

Comment:
@aklkv does this imply that we lose the ability to do these sorts of patches?

Reply:
No no, I will bring it back it's still WIP

Reply:
Ok. Ignore me then 😄